### PR TITLE
add script + update readme

### PR DESCRIPTION
**Pull request by spencer-codaio**
add temp script while CI is broken for the relase flow due to the security issue and update some docs 

cc @ekoleda-codaio @coda/packs @jonathan-codaio

### CODE REVIEW

File: scripts/update-docs.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# staging
+MK_DOCS_SITE_URL="https://staging.coda.io/packs/build/latest/" make build-mkdocs
+make publish-docs-staging FLAGS=--forceUpload
+
+# prod
+MK_DOCS_SITE_URL="https://coda.io/packs/build/latest/" make build-mkdocs
+make publish-docs-prod FLAGS=--forceUpload
+
+echo "\nFinishing pushing to all environments!"
```
Comment (jonathan-codaio) [inline scripts/update-docs.sh:18]: I would echo in between each just to checkpoint progress.

**Review by jonathan-codaio — APPROVED** [review]

File: scripts/update-docs.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# This is temporary script to update the docs while our CI is disabled due to 
+# security.
```
Comment (jonathan-codaio) [inline scripts/update-docs.sh:3]: Would avoid calling too much attention in a public repo.

Comment (spencer-codaio) [reply scripts/update-docs.sh:3]: ah right i can jst remove

File: scripts/update-docs.sh (hunk 1 above)
Comment (spencer-codaio) [reply scripts/update-docs.sh:15]: sounds good -- there is output but sounds right to make it explicit

**Pull request merged**